containers/groovy-examples/docker-compose.yml

```diff
@@ -1,49 +1,35 @@
 version: "3.4"
 
 services:
-  server:
-    image: ghcr.io/deephaven/server:${VERSION:-latest}
-    expose:
-      - '8080'
+  deephaven:
+    image: ghcr.io/deephaven/server:0.16.0-groovy
+    labels:
+      - 'traefik.http.routers.deephaven.rule=Host(`localhost`)'
     volumes:
       - ./data:/data
-      - api-cache:/cache
     environment:
-      - JAVA_TOOL_OPTIONS=-Xmx4g -Ddeephaven.console.type=groovy
+      - JAVA_OPTS=-Xmx4g
 
-  web:
-    image: ghcr.io/deephaven/web:${VERSION:-latest}
-    expose:
-      - '80'
+  notebooks:
+    image: ghcr.io/devinrsmith/webdav:latest
```

Comment:
We probably want to transition this to the deephaven namespace from https://github.com/devinrsmith/deephaven-webdav

Reply:
Yep

Reply:
env controls